jekyll/_cci2/building-docker-images.md

```diff
@@ -52,22 +52,22 @@ jobs:
       # build and push Docker image
       - run: |
           TAG=0.1.$CIRCLE_BUILD_NUM
-          docker build -t cci-demo-docker:$TAG .
-          docker login -e $DOCKER_EMAIL -u $DOCKER_USER -p $DOCKER_PASS
+          docker build -t cci-demo-docker:$TAG .                          # (3)
+          docker login -e $DOCKER_EMAIL -u $DOCKER_USER -p $DOCKER_PASS   # (4)
           docker push circleci/cci-demo-docker:$TAG
 ```
 
 Let’s break down what’s happening during this build’s execution:
 
-- All commands are executed in the [primary container][primary-container].
-- Once `setup_docker_engine` is called, a new remote environment is created, and your primary container is configured to use it.
-- All docker-related commands are also executed in your primary container, but building/pushing images and running containers happens in the remote Docker Engine.
-- We use project environment variables to store credentials for Docker Hub.
+1. All commands are executed in the [primary container][primary-container].
```

Comment:
@smart-alek it's being defined at the bottom of the page using `[primary-container]: {{ site.baseurl }}/2.0/glossary/#primary-container`